### PR TITLE
Use --host_force_python=PY2 when building the Linux release

### PR DESCRIPTION
**Pull request by philwo**
This is a workaround for https://github.com/bazelbuild/bazel/issues/8443

### CODE REVIEW

**Comment by philwo** [comment]
FYI @laurentlb @brandjon @katre

**Comment by philwo** [comment]
This is a change to the pipeline config, so it does not require a Bazel cherry-pick.

When this is merged, we can use the "Rebuild" button on the failed release build (https://buildkite.com/bazel-trusted/bazel-release/builds/66) and then it should work.

File: pipelines/bazel-release.yml [hunk 1]
```diff
@@ -66,7 +66,19 @@ steps:
...
       cp bazel-bin/src/bazel output/bazel
 
-      output/bazel build -c opt --stamp --sandbox_tmpfs_path=/tmp --embed_label "\${release_name}" --workspace_status_command=scripts/ci/build_status_command.sh src/bazel scripts/packages/with-jdk/install.sh scripts/packages/debian/bazel-debian.deb scripts/packages/debian/bazel.dsc scripts/packages/debian/bazel.tar.gz bazel-distfile.zip
+      output/bazel build \
+          -c opt \
+          --stamp \
+          --sandbox_tmpfs_path=/tmp \
+          --embed_label "\${release_name}" \
+          --workspace_status_command=scripts/ci/build_status_command.sh \
+          --host_force_python=PY2 \
```
Comment (brandjon) [inline pipelines/bazel-release.yml]: Suggest adding a comment saying why we need this / what part breaks.

Comment (philwo) [reply pipelines/bazel-release.yml]: Done!

**Pull request merged**